Framework/Algorithms/test/CrossCorrelateTest.h

```diff
@@ -134,28 +192,73 @@ class CrossCorrelateTest : public CxxTest::TestSuite {
     return ws;
   }
 
-  // Initialise the algorithm and set the properties. Creates a fake workspace
-  // for the input and returns it.
-  MatrixWorkspace_const_sptr setupAlgorithm(CrossCorrelate &alg, const double xmin, const double xmax) {
+  const MatrixWorkspace_sptr makeFakeWorkspace3Peaks(const PeakShapeEnum shape) {
+    const double D_MIN{0.9};
+    const double D_MAX{2.3};
+    const double D_DELTA{0.01};
+    const int NUM_BINS = static_cast<int>((D_MAX - D_MIN) / D_DELTA);
+    const int NUM_HIST{5};
 
-    // create the workspace
-    const MatrixWorkspace_sptr inWS = makeFakeWorkspace();
+    // create the x-axis
+    BinEdges xEdges(NUM_BINS + 1, HistogramData::LinearGenerator(D_MIN, D_DELTA));
+    auto xPoints = HistogramData::Points(xEdges);
+
+    std::vector<double> TOFvalues;
+    std::transform(cbegin(xPoints), cend(xPoints), std::back_inserter(TOFvalues),
+                   [](double d) -> double { return d * 1434.66 + d * d * -1.88 + 2.25; });
```

Comment:
I know I added these values but it is probably better to define these as constants in CrossCorrelateTestData (since they are also used there). Then move the conversion of `xPoints` (which are in d-spacing) to TOF into `CrossCorrelateTestData::evaluateFunction`. That way we can have these value only defined in a single place.

`DIFC=1434.66`, `DIFA=-1.88` and `T0=2.25`